highfidelity/hifi
# Further remove dependencies of marketplaces.js on edit.js

**Test Plan:**
1. If you don't already own some, buy a few different entities (wearables and not wearables) from the Marketplace
2. Open your logs (CTRL+SHIFT+L)
3. From the "My Purchases" app, use the "REZ" or "WEAR" button to spawn those entities (in a domain in which you have permission, of course).
4. Keep an eye on your logs. Verify that you do not see any errors in the logs immediately after pressing the REZ/WEAR button
5. Open and close Create mode. Verify that you don't see any errors in the logs.
